README.rst

```diff
@@ -78,38 +78,33 @@ Cyclus is built using ``CMake``. For detailed instructions on Cyclus dependencie
 Quick Cyclus Installation
 *************************
 The quickest way to install Cyclus and its dependencies relies on using the `conda-forge` channel and the `conda` package manager within the Anaconda python environment.  The following instructions guide you through that approach.
-To install Cyclus and its dependencies onto a clean Ubuntu machine (tested on 18.04 LTS):
+To install Cyclus and its dependencies onto a clean Ubuntu machine (tested on 22.04 LTS):
 
-- Download the latest Anaconda installer for Linux at
-  ``https://www.anaconda.com/distribution/#download-section``
+- Download and install Miniconda for Linux
 
-- Move the ``.sh`` to your Home directory
 
-- In Terminal, execute the following commands:
+.. code-block:: python
 
-- ``bash Anaconda3-2019.03-Linux-x86_64.sh``
+  mkdir -p ~/miniconda3
+  wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda3/miniconda.sh
+  bash ~/miniconda3/miniconda.sh -b -u -p ~/miniconda3
+  rm -rf ~/miniconda3/miniconda.sh
 
-- ``echo 'export PATH="~/anaconda/bin:$PATH"' >> ~/.bashrc``
+- Execute the following commands to initialize Miniconda:
 
-- ``source .bashrc``
+.. code-block:: python
 
-- ``conda config --add channels conda-forge``
+  echo 'export PATH="~/miniconda3/bin:$PATH"' >> ~/.bashrc
+  source ~/.bashrc
 
-- ``conda create -n cyclus python=3.11``
+- Create a new environment and install the Cyclus package
 
-- ``conda activate cyclus``
+.. code-block:: python
 
-- ``conda install -y gxx_linux-64 gcc_linux-64 cmake make git glib libxml2 libxmlpp-4.0 liblapack pkg-config coincbc boost-cpp hdf5 sqlite pcre setuptools pytest pytables pandas jinja2 cython websockets pprintpp pip``
-
-- ``conda install -y --force-reinstall libsqlite``
-
-- Use ``sudo apt install`` to install and configure git
-
-- Clone the Cyclus repository by running ``git clone https://github.com/cyclus/cyclus.git``
-
-- Navigate to the folder containing Cyclus
-
-- Run the command ``python install.py``
+  conda config --add channels conda-forge
+  conda create -n cyclus python=3.11
```

Comment:
For some reason our conda-forge package was not published for Python 3.12, so this needs to be pinned to 3.8, 3.9, 3.10, or 3.11.  I will look into why that is happening

Reply:
The conda-forge bot figured this out before I even had a chance to check it out.  Should work with python3.12 once the new conda-forge package is published but I will confirm tomorrow

Reply:
Can confirm these instructions work without pinning python.